docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -92,41 +112,130 @@ To enable your application to use the media controller functionality:
      This guide uses a global variable for the handle.
 
 
-## Updating and Retrieving Information
+## Retrieving Application list
```

Comment:
@haejeongkim 
Headings follow title case,
Please accommodate this change: 
## Retrieving Application List

Reply:
Done